### PR TITLE
Adds tags to doc pages

### PR DESCRIPTION
**Pull request by samhita-alla**
Signed-off-by: Samhita Alla <aallasamhita@gmail.com>

# TL;DR
This PR adds relevant tags to doc pages. 

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
Fixes https://github.com/flyteorg/flyte/issues/3078

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

File: docs/source/plugins/deck.rst
Comment (cosmicBboy) [inline docs/source/plugins/deck.rst:7]: should we add a tag "DataVisualization"?

**Comment by cosmicBboy** [comment]
this needs a +1 from code owner @wild-endeavor @eapolinario @pingsutw

**Review by pingsutw — APPROVED** [review]

**Pull request merged**
